### ISSUE TITLE
[BUG] Right Click Bug

### ISSUE THREAD

**Issue by ginsm** [ginsm/forgotten-graves#8]
**Describe the bug**
Sometimes right clicking a grave, as the owner of the grave with an empty hand, will fail to retrieve it.

**Minecraft Version**
1.17.x

**Forgotten Graves Version**
v1.1.0

**To Reproduce**
1. Kill your character
2. Ensure the area unloads, or simply relog
3. You should no longer be able to retrieve the grave using right click

**Expected behavior**
Right clicking a grave, as the owner of the grave with an empty hand, should always retrieve it (provided it's allowed within the configuration).

**Additional context**
Something happens when the grave unloads.. my initial suspicion is that there's a syncing issue happening.. but I'll need to do more testing.

**Comment by ginsm** [comment]
This bug seems to not be occurring in the most recent iteration of the mod. I'm going to leave it open and do further testing.

**Comment by ginsm** [comment]
Alright, I did everything I could think of to cause this issue to occur (even all at the same time) and it didn't occur:

1. I TPed to `2500 80 2500` (well outside spawn chunks) and typed `/kill`
2. I proceeded to go to the end and go through the portal back to the overworld
3. I then logged out
4. I restarted the server
5. And finally, I TPed back to `2500 80 2500` and was able to access the grave through `ON_USE`.

I'm going to close this for now.
